patternfly_old/Alignment/examples/alignment-center-align-example.hbs

```diff
@@ -1 +1 @@
-{{#> alignment pf-c-alignment--Modifier="pf-m-center" }} content {{/alignment}}
\ No newline at end of file
+{{#> alignment alignment--modifier="pf-m-center"}} content {{/alignment}}
```

Comment:
I'm curious about the use of `alignment--modifier`, considering the next two examples use `pf-c-alignment--Modifier`. Can you please clarify why they are different?

Reply:
FYI we have an issue to get rid of patternfly_old - https://github.com/patternfly/patternfly-next/issues/770